ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -153,25 +153,9 @@ private void onBlockAdded(final BlockAddedEvent blockAddedEvent) {
               });
     } else {
 
-      LOG.trace("Not ready for import blocks found for {}", newBlock.getHeader().getNumber());
-
-      pendingBlocksManager
-          .lowestAnnouncedBlock()
-          .map(ProcessableBlockHeader::getNumber)
-          .ifPresent(
-              minAnnouncedBlockNumber -> {
-                long distance =
-                    minAnnouncedBlockNumber
-                        - protocolContext.getBlockchain().getChainHeadBlockNumber();
-                LOG.trace(
-                    "Found lowest announced block {} with distance {}",
-                    minAnnouncedBlockNumber,
-                    distance);
-                if (distance < config.getBlockPropagationRange().upperEndpoint()
-                    && minAnnouncedBlockNumber > newBlock.getHeader().getNumber()) {
-                  retrieveMissingAnnouncedBlock(newBlock.getHeader().getNumber() + 1);
-                }
-              });
+      LOG.trace("No ready for import blocks found for {}", newBlock.getHeader().getNumber());
```

Comment:
Not ready (not No ready)

Reply:
the code, mean that there are no blocks that can imported, probably I will rephrase the log since in not clear